tools/gen_allocs.c

```diff
@@ -270,6 +270,13 @@ if ( tag == 1 )
 	   fprintf(fp,"  num_bytes_allocated = num_bytes_allocated + &\n(%s) * %cWORDSIZE\n",
                    array_size_expression("", "(", -1, t2, p, post_for_count, "model_config_rec%"),
                    tchar) ;
+	   fprintf(fp,"  nba = &\n(%s) * %cWORDSIZE\n",
+                   array_size_expression("", "(", -1, t2, p, post_for_count, "model_config_rec%"),
+                   tchar) ;
+           fprintf(fp,"#if ( SHOW_ALL_VARS_USED == 1 )\n") ;
+           fprintf(fp,"  WRITE(message_string,fmt='(a,i8)') '%s: bytes = ',nba\n", fname) ;
```

Comment:
Wouldn't this format only allow us to see the size of variables that were <100 MB?